app/representers/api/image_representer.rb

```diff
@@ -2,13 +2,16 @@
 
 class Api::ImageRepresenter < Roar::Decorator
   include Roar::JSON
+  include HalApi::Representer::FormatKeys
 
-  property :url
+  property :href
```

Comment:
I want to remove `url` from this API entirely in favor of `href` ... but will have to update the Castle code that scrapes Feeder first.